contrib/win32/install/install/product.wxs

```diff
@@ -0,0 +1,170 @@
+    https://support.firegiant.com/hc/en-us/articles/230912227-Control-feature-states-for-silent-install-
+    -->
+    <Feature Id="ProductFeature" Title="OpenSSH" Level="1" Display="expand" AllowAdvertise="no">
+			<ComponentGroupRef Id="CG_Main" />
+      <ComponentGroupRef Id="CG_Main_Symbols"/>
+      <Feature Id="F_Client" Title="Client programs" Level="1" Absent="allow" AllowAdvertise="no">
+        <Condition Level="10000">DISABLE_CLIENT</Condition>
+        <ComponentGroupRef Id="CG_Client" />
+        <Feature Id="F_Client_Symbols" Title="Client programs synbols" Level="10000" Absent="allow" AllowAdvertise="no">
+          <Condition Level="1">ENABLE_CLIENTSYMBOLS</Condition>
+          <ComponentGroupRef Id="CG_Client_Symbols"/>
+        </Feature>
+      </Feature>
+      <Feature Id="F_Server" Title="Server programs" Level="10000" Absent="allow" AllowAdvertise="no">
+        <Condition Level="1">ENABLE_SERVER</Condition>
+        <ComponentGroupRef Id="CG_Server" />
+        <Feature Id="F_SshdConfig" InstallDefault="followParent" Display="hidden" AllowAdvertise="no">
+          <ComponentGroupRef Id="CG_SshdConfig" />
+          <ComponentRef Id="C_PURGE_SSH_LOGS" />
+          <ComponentRef Id="C_PURGE_SSH" />
+        </Feature>
+        <Feature Id="F_Server_Symbols" Title="Server programs synbols" Level="10000" Absent="allow" AllowAdvertise="no">
+          <Condition Level="1">ENABLE_SERVERSYMBOLS</Condition>
+          <ComponentGroupRef Id="CG_Server_Symbols"/>
+        </Feature>
+      </Feature>
+      <Feature Id="F_Scripts" Title="Management scripts" Level="10000" Absent="allow" AllowAdvertise="no">
+        <Condition Level="1">ENABLE_SCRIPTS</Condition>
+        <ComponentGroupRef Id="CG_Scripts" />
+      </Feature>
+    </Feature>
+	</Product>
+
+	<Fragment>
+		<Directory Id="TARGETDIR" Name="SourceDir">
+      <Directory Id="$(var.PROGRAMFILESFOLDER)">
+  			<Directory Id="INSTALLFOLDER" Name="OpenSSH" />
+			</Directory>
+      <Directory Id="CommonAppDataFolder">
+        <Directory Id="SSHDFOLDER" Name="ssh">
+          <Component Id="C_PURGE_SSH" Guid="a2a04863-6287-4650-bc48-ed857d86c23d">
+            <RemoveFile Id="RemoveSshFiles" Name="*" On="uninstall"/>
+            <RemoveFolder Id="RemoveSshFolder" On="uninstall"/>
+          </Component>
+          <Directory Id="SSHDLOGFOLDER" Name="logs">
+            <Component Id="C_PURGE_SSH_LOGS" Guid="2311dc7e-0580-4c35-9acf-9cc381e6a90e">
+              <RemoveFile Id="RemoveLogFiles" Name="*" On="uninstall"/>
+              <RemoveFolder Id="RemoveLogsFolder" On="uninstall"/>
+            </Component>
+          </Directory>
+        </Directory>
+      </Directory>
+    </Directory>
+	</Fragment>
+
+	<Fragment>
+		<ComponentGroup Id="CG_Main" Directory="INSTALLFOLDER">
+      <?include components/c_main.wxi ?>
+		</ComponentGroup>
+    <ComponentGroup Id="CG_Main_Symbols" Directory="INSTALLFOLDER">
+      <?include components/c_main_symbols.wxi ?>
```

Comment:
Rather than includes, just use a `ComponentRef` or `ComponentGroupRef` like you did above for things defined here, but then just link those .wxs files in. Just add them as `<Compile>` items and WiX does the rest. It links them all together like a C++ project would link .obj files created from .cpp files - in fax, WiX build was modeled after this: .wxs -> .wixobj -> .msi + .wixpdb.

Reply:
I checked the WXS source code in the PowerShell GitHub repo and I understood the way to go. Makes the installer even simpler 👍🏼

Reply:
Removed the includes